lib/cfg-grammar.y

```diff
@@ -1299,6 +1300,7 @@ threaded_dest_driver_batch_option
 threaded_dest_driver_workers_option
         : KW_WORKERS '(' positive_integer ')'  { log_threaded_dest_driver_set_num_workers(last_driver, $3); }
         | KW_WORKER_PARTITION_KEY '(' template_content ')' { log_threaded_dest_driver_set_worker_partition_key_ref(last_driver, $3); }
+        | KW_FLUSH_ON_WORKER_KEY_CHANGE '(' yesno ')'	{ log_threaded_dest_driver_set_flush_on_worker_key_change(last_driver, $3); }
```

Comment:
Why is this user configurable?wouldn't it make more sense to delegate this to the driver?

Reply:
We decided to publish this as the HTTP destination is considered a final class, but it's general enough to work in both modes.

Reply:
couldn't we make it a http() only option then?